tests/rados/test_stretch_n-az_netsplit_scenarios.py

```diff
@@ -0,0 +1,286 @@
+        )
+
+        log.debug(
+            "Checking is the cluster is marked degraded and "
+            "operating in degraded mode post Netsplit b/w data sites"
+        )
+
+        log.debug("sleeping for 4 minutes before starting writes.")
+        time.sleep(600)
+
+        # log cluster health
+        rados_obj.log_cluster_health()
+
+        # Starting checks to see availability of cluster during netsplit scenario
+        # perform rados put to check if write ops is possible
+        pool_obj.do_rados_put(client=client_node, pool=pool_name, nobj=200, timeout=100)
+        # rados_obj.bench_write(pool_name=pool_name, rados_write_duration=100)
+
+        log.debug("sleeping for 4 minutes for the objects to be displayed in ceph df")
+        time.sleep(600)
+
+        # Getting the number of objects post write, to check if writes were successful
+        pool_stat_final = rados_obj.get_cephdf_stats(pool_name=pool_name)
+        log.debug(pool_stat_final)
+        final_objects = pool_stat_final["stats"]["objects"]
+        log.debug(
+            f"Final number of objects on the pool : {pool_name} is {final_objects}"
+        )
+
+        # Objects should be more than the initial no of objects
+        if int(final_objects) <= int(init_objects):
+            log.error(
+                "Write ops should be possible, number of objects in the pool has not changed"
+            )
+            raise Exception(
+                f"Pool {pool_name} has {pool_stat['stats']['objects']} objs"
+            )
+
+        log.info(
+            f"Successfully wrote {int(final_objects) - int(init_objects)} on pool {pool_name} in degraded mode\n"
+            f"Proceeding to remove the IPtable rules and recover the cluster from degraded mode"
+        )
+
+        time.sleep(5)
+
+        # Starting to flush IP table rules on all hosts
+        for host in cluster_nodes:
+            log.debug(f"Proceeding to flush iptable rules on host : {host.hostname}")
+            host.exec_command(sudo=True, cmd="iptables -F", long_running=True)
+            log.debug(
+                "Observed that just IP tables flush did not work to bring back the nodes to cluster."
+                f"rebooting the nodes post testing. Rebooting node : {host.hostname}"
+            )
+            host.exec_command(sudo=True, cmd="reboot")
+        log.debug("Sleeping for 30 seconds...")
+        time.sleep(30)
+
+        log.info("Proceeding to do checks post Stretch mode netsplit scenarios")
+
+        if not rados_obj.run_pool_sanity_check():
+            log.error("Checks failed post Site Netsplit scenarios")
+            raise Exception("Post execution checks failed on the Stretch cluster")
+
+    except Exception as err:
+        log.error(f"Hit an exception: {err}. Test failed")
+        log.debug(
+            "Test case expected to fail until bug fix : https://bugzilla.redhat.com/show_bug.cgi?id=2265116"
```

Comment:
BZ#2265116 is closed with the duplicate of BZ#2263480 and this bug is closed with the  insufficient data.We need to re-open or raise a new bug.